main/src/flix/runtime/library/MonoStringBuilder.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2019 Stephen Tetley
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package flix.runtime.library;
+
+import java.lang.StringBuilder;
+import java.math.BigInteger;
+
+/**
+ * Wrap Java's StringBuilder, provide a small, monomorphic API.

```

Comment:
A wrapper class for StringBuilder that provides concrete, non-overloaded methods.